src/content/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data.mdx

```diff
@@ -74,7 +74,7 @@ Tags are automatically applied to your monitored entities in some cases, from th
 
 Tags from these sources can be removed or changed only by modifying the source of the tag, not via the UI or API.
 
-You can also programmatically automate tags using our [API](/docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial/) or the [New Relic CLI](https://developer.newrelic.com/automate-workflows/5-mins-tag-resources).
+You can also programmatically automate tags using our [API](/docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial/), the [New Relic CLI](https://developer.newrelic.com/automate-workflows/5-mins-tag-resources) or the [New Relic Terraform Provider](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/entity_tags).
```

Comment:
should "Provider" be lowercase?

Reply:
Also, I think we need a comma before the "or"

Reply:
@paperclypse good catch on the Oxford comma! I just fixed this. Shall I make "Provider" lower case even though it's capitalized in GitHub?

Reply:
That's what makes sense to me